RELEASING.md

```diff
@@ -1,12 +1,55 @@
  4. A new branch and PR will automatically be created
-3. Wait until PR is approved (don't merge yet)
+3. Wait until PR is approved (don't merge yet) and pull branch from origin (to make sure you've got all the changes locally)
 4. When the PR is approved, approve the hold job created in CircleCI. CircleCI will create a tag for the version. Alternatively, you can tag the last commit in the release branch and push it to the repository.
-5. The rest will be performed automatically by CircleCI. If the automation fails, you can run the `scripts/create-unity-package.sh` manually to create the packages and create a github release manually and add the resulting `unitypackage` files.
+5. The rest will be performed automatically by CircleCI. If the automation fails, you can revert to manually calling `bundle exec fastlane deploy`.
 6. After that, you can merge the release PR to main and merge the bump to the next snapshot version PR right after.
+
+Hotfix Releases
+=========
+Sometimes you might need to release a patch on a version that's not the latest. Or sometimes there might have been commits on main that shouldn't be released, but the latest version has a big bug. For example, let's say the last version is 4.0.0, but there's a bug on 3.9.0 that needs to be released as a 3.9.1:
+
+1. Open a PR with the fix, merge it to main.
+1. Jump to 3.9.0 and create a new branch bump/3.9.0. Push this branch since it will be used as base for the PR you will open in the next steps.
+1. Run `bundle exec fastlane bump` as you would do with any release. This will create a new PR `release/3.9.1`. Make sure the base branch of the PR is pointing to `release/3.9.0` and that the PR is labeled with **DO NOT MERGE** (add it to the title too so it's not merged by mistake)
+1. Locally, while in branch the release branch (`release/3.9.1` in this case), cherry pick the changes with your fix that you just merged to `main` using `git cherry-pick <sha_of_the_squashed_commit_in_main>`. Fix any conflicts if there are any. Push the cherry picked commit to the remote branch `release/3.9.1`.
+1. When the PR is approved, **DO NOT MERGE IT**. Approve the hold job in CircleCI as you would do for any other release. If there is no hold job because the release is older than when we introduced that job, manually tag the last commit in `release/3.9.1` with `3.9.1`.
+1. CircleCI will start the deployment process
+1. Close the PR after the release has been completed and delete both `release/3.9.0` and `release/3.9.1` branches.
+1. Remember to edit the CHANGELOG.md in `main` to include the version that has been just released
+
+Fixing timeouts when closing repository
```

Comment:
I think this applies to android only? We should remove this section from here.

Reply:
LOOOL yeah, totaly copy/pasta, good catch